src/EmployerPR/SFA.DAS.EmployerPR/Domain/RoatpProvider/GetRoatpProvidersResponse.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace SFA.DAS.EmployerPR.Domain.RoatpProvider;
+public class GetRoatpProvidersResponse
+{
+    public List<Application.Queries.GetRoatpProviders.RoatpProvider> RegisteredProviders { get; set; }
```

Comment:
I still do not see how this is being useful, it looks same as GetRoatpProvidersQueryResult

Reply:
sorry I got mixed up, this isn't useful, you're right :)
removed